src/arizona/index.ts

```diff
@@ -0,0 +1,37 @@
+import cheerio from "cheerio";
+import axios from "axios";
+import { timeout } from "../helpers";
+
+(async () => {
+	const startingId = 2177292;
+	for (let i = 0; i <= 20; i++ ) {
+        await getDetails(startingId + i);
+        //Longer timeout needed because of DDOS protection from website
+		await timeout(5000);

```

Comment:
```suggestion
	await timeout(5000);

```

Let's clean up the formatting while we remove the hubzu thingy.